src/docs/release-notes/R19/R19.mdx

```diff
@@ -0,0 +1,35 @@
+### Figma Stickersheet
+
+-   New Design Pattern Stickersheet (TODO: Link) with examples of all of our [Design Patterns](/patterns).
```

Comment:
1. I personally try to call them "Design Pattern Library" and "Component Sticker Sheet" (with space) on the design end. Although if we go with my route, we will have to change the R18 release notes too.
2. It is not technically a _new_ Figma file, since we sorta released it last quarter (with only three patterns). 
3. In R18, this content falls under "design patterns" section

Reply:
You're right. I thought we hadn't published it yet because we were waiting for all of them to be finished. I'll update this. I'm moving the release notes to a separate PR from the dependency updates.